client/services/code_evaluators/PythonPrereqCheckService.js

```diff
@@ -164,6 +192,14 @@ tie.factory('PythonPrereqCheckService', [
             PREREQ_CHECK_TYPE_GLOBAL_CODE, null, starterCode);
         }
 
+        if (checkInvalidSystemClassCalls(code) === SYSTEM_CALL) {
```

Comment:
Would expect something more like: `if (hasInvalidSystemClassCalls(code))`, and `if (hasInvalidAuxiliaryClassCalls(code))`. Then you don't need to pass those strings around.